lang/llvm-3.4/Portfile

```diff
@@ -63,7 +63,7 @@ if {${subport} eq "llvm-${llvm_version}"} {
     }
 }
 
-if {${os.platform} eq "darwin" && ${os.major} < 11 && ${cxx_stdlib} eq "libc++"} {
+if {${os.platform} eq "darwin" && ${os.major} < 12 && ${cxx_stdlib} eq "libc++"} {
```

Comment:
I don't think this is necessary, as 3.4 is only needed to build libc++ and dependencies, and 10.7 already has it.

Reply:
On Mac OS X 10.7,
* icu depends on clang-3.4
* clang-3.4 depends on libxml2
* libxml2 depends on icu
* ...

Reply:
That shouldn't be the case, clang-3.4 should depend on libxml2-bootstrap.

Reply:
Oh on 10.7 yes. Because it doesn't have a C++11 compiler despite having libc++.

Reply:
I still don't like forcing libstdc++ on a platform that has libc++. How about adjusting libxml2-bootstrap to use icu58 on >= 10.7 and just using that in llvm-3.4?

Reply:
See 58c748e. I'd suggest this change to llvm-3.4 now:
```diff
index 3fa35d05d1f..37ca193c994 100644
--- a/lang/llvm-3.4/Portfile
+++ b/lang/llvm-3.4/Portfile
@@ -63,13 +63,16 @@ if {${subport} eq "llvm-${llvm_version}"} {
     }
 }
 
-if {${os.platform} eq "darwin" && ${os.major} < 11 && ${cxx_stdlib} eq "libc++"} {
-    configure.cxx_stdlib    libstdc++
-    # Have to also use bootstrap versions of deps that use libstdc++ in
-    # order to be able to build libc++.
-    depends_lib-replace port:libxml2 port:libxml2-bootstrap \
-                        port:python27 port:python27-bootstrap \
-                        port:ncurses port:ncurses-bootstrap
+if {${os.platform} eq "darwin" && ${os.major} <= 11 && ${cxx_stdlib} eq "libc++"} {
+    if {${os.major} < 11} {
+        configure.cxx_stdlib    libstdc++
+        # Have to also use bootstrap versions of deps that use libstdc++ in
+        # order to be able to build libc++.
+        depends_lib-replace port:python27 port:python27-bootstrap \
+                            port:ncurses port:ncurses-bootstrap
+    }
+    # libxml2 depends on icu which needs a C++11 compiler
+    depends_lib-replace     port:libxml2 port:libxml2-bootstrap
     configure.cppflags-prepend  -I${bootstrap_prefix}/include
     configure.ldflags-prepend   -L${bootstrap_prefix}/lib
     configure.env-append    PATH=${bootstrap_prefix}/bin:$::env(PATH)
```

Reply:
This change should now be made.